src/Sparrow/LowMemory/MemoryInformation.cs

```diff
@@ -425,73 +419,120 @@ private static unsafe MemoryInfoResult GetMemoryInfoWindows()
             // Malformed SMBIOS data may indicate a problem with the user's computer.
             var fetchedInstalledMemory = GetPhysicallyInstalledSystemMemory(out var installedMemoryInKb);
 
+            var sharedClean = GetSharedCleanInBytes(process);
             SetMemoryRecords((long)memoryStatus.ullAvailPhys);
-
+            
             return new MemoryInfoResult
             {
                 TotalCommittableMemory = new Size((long)memoryStatus.ullTotalPageFile, SizeUnit.Bytes),
                 CurrentCommitCharge = new Size((long)(memoryStatus.ullTotalPageFile - memoryStatus.ullAvailPageFile), SizeUnit.Bytes),
                 AvailableMemory = new Size((long)memoryStatus.ullAvailPhys, SizeUnit.Bytes),
-                AvailableWithoutTotalCleanMemory = new Size((long)memoryStatus.ullAvailPhys, SizeUnit.Bytes),
-                SharedCleanMemory = Size.Zero,
+                AvailableWithoutTotalCleanMemory = new Size((long)memoryStatus.ullAvailPhys + sharedClean, SizeUnit.Bytes),
+                SharedCleanMemory = new Size(sharedClean, SizeUnit.Bytes),
                 TotalPhysicalMemory = new Size((long)memoryStatus.ullTotalPhys, SizeUnit.Bytes),
                 InstalledMemory = fetchedInstalledMemory ?
                     new Size(installedMemoryInKb, SizeUnit.Kilobytes) :
                     new Size((long)memoryStatus.ullTotalPhys, SizeUnit.Bytes),
-                MemoryUsageRecords = new MemoryInfoResult.MemoryUsageLowHigh
+                WorkingSet = new Size(process?.WorkingSet64 ?? 0, SizeUnit.Bytes)
+            };
+        }
+
+        public static long GetSharedCleanInBytes(Process process)
+        {
+            if (process == null)
+                return 0;
+            
+            var mappedDirty = 0L;
+            foreach (var mapping in NativeMemory.FileMapping)
+            {
+                var fileMappingInfo = mapping.Value.Value;
+                var fileType = fileMappingInfo.FileType;
+                if (fileType == NativeMemory.FileType.Data)
+                    continue;
+
+                var totalMapped = GetTotalMapped(fileMappingInfo);
+                if (fileType == NativeMemory.FileType.ScratchBuffer)
                 {
-                    High = new MemoryInfoResult.MemoryUsageIntervals
-                    {
-                        LastOneMinute = new Size(HighLastOneMinute, SizeUnit.Bytes),
-                        LastFiveMinutes = new Size(HighLastFiveMinutes, SizeUnit.Bytes),
-                        SinceStartup = new Size(HighSinceStartup, SizeUnit.Bytes)
-                    },
-                    Low = new MemoryInfoResult.MemoryUsageIntervals
+                    // for scratch buffers we have the allocated size
+                    var allocated = fileMappingInfo.GetAllocatedSizeFunc?.Invoke() ?? 0;
```

Comment:
If the `GetAllocatedSizeFun` isn't set, return `totalMapped`, not `0`

Reply:
done